frontend/src/pages/modelRegistry/screens/RegisterModel.tsx

```diff
@@ -0,0 +1,319 @@
+import React from 'react';
+import {
+    ActionGroup,
+    Alert,
+    AlertActionCloseButton,
+    Breadcrumb,
+    BreadcrumbItem,
+    Button,
+    Form,
+    FormGroup,
+    FormSection,
+    HelperText,
+    HelperTextItem,
+    InputGroupItem,
+    InputGroupText,
+    Radio,
+    Stack,
+    StackItem,
+    Text,
+    TextInput,
+    Title,
+} from '@patternfly/react-core';
+import { useParams } from 'react-router';
+import { Link } from 'react-router-dom';
+import ApplicationsPage from '~/pages/ApplicationsPage';
+import useRegistrModelData from './useRegistrModelData';
+
+const RegisterModel: React.FC = () => {
```

Comment:
Since we have 2 new files specific to the RegisterModel screen, let's create a subdirectory here `RegisterModel` and move `RegisterModel.tsx` and `useRegisterModelData.ts` into it. We can also put a utils file for the API call in there in the next PR.